best-practices/tagging/index.html

```diff
@@ -583,7 +612,8 @@ <h4>Tables</h4>
 					</dd>
 					<dt>Attributes</dt>
 					<dd>
-						<p>[^scope^], [^headers^], [^axis^]</p>
+						<p>The attributes [^scope^] and [^headers^] must be used with an [^id^] to properly identify the relationship between the cells of the table.</p>
```

Comment:
The way I understood it is that `headers` must be used in conjunction with `id`, and that this is an alternative to using the `scope` attribute. I'm not sure it makes sense to use both at the same time. Haven't seen any examples like that.

Hmm, too bad that `axis` is deprecated. I liked it.

Reply:
I didn't think of that. I had a difficult time searching for information about `headers` because it kept thinking I meant `th`. Most people seem to use `scope`. I'm going to edit these tables to just use `scope` and then we can add a complex example later.